frontend/src/scenes/organization/Settings/VerifiedDomains/SSOSelect.tsx

```diff
@@ -5,7 +5,7 @@ import { SSO_PROVIDER_NAMES } from 'lib/constants'
 import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
 import { SSOProvider } from '~/types'
 
-interface SSOSelectInterface {
+export interface SSOSelectInterface {
```

Comment:
Drive-by fix, fixing a TypeScript error in some test code (which therefore doesn't trigger CI failures, but does still show up in my editor)